_includes/manuals/3.11/1.2_release_notes.md

```diff
@@ -4,19 +4,206 @@ This section will be updated prior to the next release.
 
 ### Headline features
 
+#### Greatly decreased JavaScript size for plugins
+
+Previously, Foreman's whole JavaScript bundle was duplicated in every plugin.
+Now a separate bundle is generated that Foreman and each plugin can reuse.
+In most plugins, we saw a 2 to 3 MB reduction in size and depending on the number of plugins, this can save a significant amount of space.
+While Foreman does compress and cache these JavaScript bundles, they still had to be loaded all the time.
+
+For more details, see https://projects.theforeman.org/issues/37252 and https://projects.theforeman.org/issues/37391.
+
+#### Running Foreman on {EL} 9 is fully supported
```

Comment:
EL is not a macro that gets properly expanded here

Reply:
Copy paste from `foreman-documentation`. What can I used instead?

Reply:
I'd use "Enterprise Linux", in `foreman-documentation` we expand it to either "Enterprise Linux" or "Red Hat Enterprise Linux", depending on the type of build (upstream vs satellite), but here we don't have to do the same differentiation

Reply:
Ok addressed that